go-controller/pkg/cni/cni.go

```diff
@@ -132,6 +132,7 @@ func (pr *PodRequest) cmdAdd(kubeAuth *KubeAPIAuth, podLister corev1listers.PodL
 			return nil, err
 		}
 	} else {
+		podInterfaceInfo.PodUID = pr.PodUID
```

Comment:
@cathy-zhou might as well do this for both cases?

Reply:
@dcbw I've make changes according to your commets and repushed, can you please take a look?